### PR TITLE
Add Capability support to play kube

### PR DESCRIPTION
**Pull request by haircommander**
as well as fix a generate bug where the capabilites were duplicated
fixes https://github.com/containers/libpod/issues/3689

### CODE REVIEW

**Comment by mheon** [comment]
I think your third commit might need to be squashed :smile:

File: pkg/adapter/pods.go [hunk 1]
```diff
@@ -682,6 +682,15 @@ func kubeContainerToCreateConfig(ctx context.Context, containerYAML v1.Container
 		if containerYAML.SecurityContext.AllowPrivilegeEscalation != nil {
 			containerConfig.NoNewPrivs = !*containerYAML.SecurityContext.AllowPrivilegeEscalation
 		}
+
+	}
+	if caps := containerYAML.SecurityContext.Capabilities; caps != nil {
+		for _, capability := range caps.Add {
```
Comment (mheon) [inline pkg/adapter/pods.go]: I think lint is going to want this changed to `containerConfig.CapAdd = append(containerConfig.CapAdd, caps.Add...)`

Comment (haircommander) [reply pkg/adapter/pods.go]: they shouldn't be compatible types though

Comment (mheon) [reply pkg/adapter/pods.go]: Ahhh, yeah the conversion should break that one.

**Comment by haircommander** [comment]
> I think your third commit might need to be squashed smile

curse my hasty pushing!

**Comment by mheon** [comment]
LGTM once tests go green

**Comment by TomSweeneyRedHat** [comment]
4th times the charm?
LGTM assuming happy tests.

**Pull request merged**